Sources/SwiftParserDiagnostics/ParserDiagnosticMessages.swift

```diff
@@ -156,6 +156,12 @@ extension DiagnosticMessage where Self == StaticParserError {
   public static var expectedSequenceExpressionInForEachLoop: Self {
     .init("expected Sequence expression for for-each loop")
   }
+  public static var extraTokensAtTheEndOfSourceLocationDirective: Self {
+    .init("extra tokens at the end of #sourceLocation directive")
```

Comment:
Should we make this consistent with the `#else` diagnostic?

```suggestion
    .init("extra tokens following the #sourceLocation directive")
```